### PR TITLE
:pencil: Home changes

### PR DESCRIPTION
**Pull request by williln**
Related to #204. Depends on https://github.com/djangocon/2018.djangocon.us/pull/203

Changes proposed in this PR:

- Reflect tickets are sold out but waiting list exists
- Reflect sponsorship closed but can inquire for next year 
- Adds childcare info to various places 
- Reflects hotel block sold out more explicitly. 
- Reflect food accommodations at meals 
- Links to State of Django panel (hence dependency on that PR) 
- Replaces Tickets button in nav to button that links to CoC for emergencies.

### CODE REVIEW

File: _includes/nav.html [hunk 1]
```diff
@@ -45,7 +45,9 @@
 </li>
 <li><a href="/news/">News</a></li>
 <li class="has-button">
-  <a class="button hollow theme-festival" href="{{ site.ticket_link }}">
-    Tickets On Sale!
+  <a class="button hollow theme-festival" href="/conduct/">
```
Comment (jefftriplett) [inline _includes/nav.html]: This is a great idea, but we aren't there yet with tutorials still selling. Can we remove / queue this up for a few days before?

Comment (williln) [reply _includes/nav.html]: @jefftriplett Should we change it to "Tutorials still on sale" then?

Comment (jefftriplett) [reply _includes/nav.html]: Maybe "Limited Tickets Left" or something better worded instead?

File: _layouts/home.html [hunk 2]
```diff
@@ -98,8 +98,8 @@ <h2 class="contrast-fix">A conference with heart</h2>
           class="card-icon"
           />
         <div class="card-section">
-          <h3 class="card-title">Sponsors!</h3>
-          <p>We can't put on DjangoCon US without you. Go ahead and <strong><a href="mailto:{{site.sponsors_email}}">drop us a line</a></strong> so we can talk to you about how we can make it easy for you to sponsor us.</p>
+          <h3 class="card-title">State of Django</h3>
+          <p>We're presenting a panel on the <a href="/talks/state-of-django/">State of Django</a>: the codebase and the community. Join on on Wednesday afternoon to hear from Django contributors, the Django Software Foundation, Django Girls, and DEFNA.</p>
```
Comment (kennethlove) [inline _layouts/home.html]: `on on` should be `us on`?

**Comment by williln** [comment]
@kennethlove @jefftriplett Tickets link now says "Limited Tickets Available", grammar mistake in State of Django panel corrected, and link to state of django panel removed until we have those details merged. Merge this at will!

**Pull request merged**